baselines/Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -367,10 +368,10 @@ This section addresses Drive for Desktop, a feature that enables users to intera
 ### Policies
 
 #### GWS.DRIVEDOCS.6.1v0.2
-Agencies SHOULD either disable Google Drive for Desktop or only allow Google Drive for Desktop on authorized devices.
+Google Drive for Desktop SHOULD only be enabled for authorized devices.
```

Comment:
Maybe this one is just personal preference, but I think moving the "only" makes it read better.
```suggestion
Google Drive for Desktop SHOULD be enabled only for authorized devices.
```